introduction.md

```diff
@@ -2,4 +2,5 @@ my name is maureen theisen
 my favorite codeschool classes are javascript
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?